test/integration/freetext_editor_spec.js

```diff
@@ -45,17 +46,58 @@ const copyPaste = async page => {
   await promise;
 };
 
-const clearAll = async page => {
+const selectAll = async page => {
   await page.keyboard.down("Control");
   await page.keyboard.press("a");
   await page.keyboard.up("Control");
-  await page.waitForTimeout(10);
+  await page.evaluate(
+    () => !!document.querySelector(".freeTextEditor:not(.selectedEditor)")
+  );
+};
+
+const clearAll = async page => {
+  await selectAll(page);
   await page.keyboard.down("Control");
   await page.keyboard.press("Backspace");
   await page.keyboard.up("Control");
   await page.waitForTimeout(10);
```

Comment:
In recent intermittent-fixes done by Tim the use of this sort of *very short* timeout was identified as a potential issue on slower hardware.
Any chance that we could await *something* rather than relying on a timeout here?